src/Line.js

```diff
@@ -127,7 +130,7 @@ class Line {
       .attr('class', 'xkcd-chart-line')
       .attr('d', (d) => theLine(d.data))
       .attr('fill', 'none')
-      .attr('stroke', (d, i) => (this.options.dataColors
+      .attr('stroke', (d, i) => (!isEmptyArray(this.options.dataColors)
```

Comment:
if `this.options` is undefined, looks the error still  exists as we are we are still trying to get a property of undefined here
`this.options.dataColors`.

May be check `this.options && this.options.dataColors` here?

Not 100 present sure as I dont got a computer nearby

Reply:
@timqian Thanks for getting back.
Correct, let me do that check and update this PR